omod/src/main/resources/messages.properties

```diff
@@ -16,16 +16,20 @@ ${project.parent.artifactId}.ccd.label=CCD Exceptions
 ${project.parent.artifactId}.details.description=Outgoing Exceptions Details
 ${project.parent.artifactId}.details.label=Exceptions Details
 ${project.parent.artifactId}.details.id.label=Id
-${project.parent.artifactId}.details.messageBody.label=Message Body
+${project.parent.artifactId}.details.messageBody.label=Message body
 ${project.parent.artifactId}.details.timestamp.label=Timestamp
 ${project.parent.artifactId}.details.failureReason.label=Failure reason
 ${project.parent.artifactId}.details.destination.label=Destination
 ${project.parent.artifactId}.details.type.label=Type
 ${project.parent.artifactId}.details.failure.label=Failure
 ${project.parent.artifactId}.details.owner.label=Owner
+${project.parent.artifactId}.details.retry.is_retried=Retried
+${project.parent.artifactId}.details.retry.last_retry_time=Last retry time
+${project.parent.artifactId}.details.retry.retry_result=Retry result
+
 ${project.parent.artifactId}.details.messageNotFound=Message not found
 
 ${project.parent.artifactId}.message.header.timestamp=Timestamp
 ${project.parent.artifactId}.message.header.destination=Destination
-${project.parent.artifactId}.message.header.user=User
-${project.parent.artifactId}.message.header.failure=Success/Failure
\ No newline at end of file
+${project.parent.artifactId}.message.header.user=Owner
+${project.parent.artifactId}.message.header.failure=Successfully retried
```

Comment:
I think you need to change the key of this property

Reply:
and add a new line at the eof